core/arch/arm/plat-stm32mp1/drivers/stm32mp1_clk.c

```diff
@@ -1286,6 +1286,24 @@ static void get_osc_freq_from_dt(void *fdt)
 	}
 }
 
+static void enable_static_secure_clocks(void)
+{
+	unsigned int idx = 0;
+	const unsigned long secure_enable[] = {
```

Comment:
This should probably be static too.

Reply:
Why do you think so? This array is used only once, at boot time.

Reply:
The data which is used to initialize this array on the stack is stored somewhere. By making it static you take this variable off the stack and at the same time give a name to the initialization data.

Reply:
ok, i understand. thanks for the explanations.